source/indexes/create-vector-search-index.txt

```diff
@@ -0,0 +1,146 @@
+.. _compass-create-vector-search:
+
+===================================
+Create an Atlas Vector Search Index
+===================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.42.1
+
+You can create Atlas Vector Search indexes using |compass|. These indexes 
```

Comment:
Would it make sense to link to the general documentation on Atlas Vector Search indexes here?
https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-overview/

Reply:
Yes!